docs/types.rst

```diff
@@ -102,6 +102,10 @@ and to send Ether (in units of wei) to an address using the ``send`` function:
     to make safe Ether transfers, always check the return value of ``send`` or even better:
     Use a pattern where the recipient withdraws the money.
 
+* ``transfer``
```

Comment:
Please check line 83, 87, etc. It should mainly explain transfer and mention send only as a special case.